README.md

```diff
@@ -39,67 +43,146 @@ Skopeo works with API V2 registries such as Docker registries, the Atomic regist
 
 Inspecting a repository
 -
-`skopeo` is able to _inspect_ a repository on a Docker registry and fetch images layers.
+`skopeo` is able to _inspect_ a repository on a container registry and fetch images layers.
 The _inspect_ command fetches the repository's manifest and it is able to show you a `docker inspect`-like
 json output about a whole repository or a tag. This tool, in contrast to `docker inspect`, helps you gather useful information about
 a repository or a tag before pulling it (using disk space).  The inspect command can show you which tags are available for the given 
 repository, the labels the image has, the creation date and operating system of the image and more.  
 
-
 Examples:
 ```sh
```

Comment:
`console` might look better than `sh`. (I haven’t tried it for this content.)

Reply:
Switched